model-client-js-test/replicated-model-test/build.gradle.kts

```
@@ -0,0 +1,23 @@
+import com.github.gradle.node.npm.task.NpmTask
+
+plugins {
+    base
+    alias(libs.plugins.node)
+}
+
+val updateDependencies = tasks.create<NpmTask>("updateDependencies") {
```

```suggestion
val updateDependencies = tasks.register<NpmTask>("updateDependencies") {
```
For task configuration avoidance.